src/wireviz/DataClasses.py

```diff
@@ -90,7 +90,7 @@ class Cable:
     length: float = 0
     color: Optional[str] = None
     wirecount: Optional[int] = None
-    shield: bool = False
+    shield: Union[bool, str] = False # False | True | color
     notes: Optional[str] = None
     colors: List[Any] = field(default_factory=list)
     color_code: Optional[str] = None
```

Comment:
Why have all attributes of the class `Connection` below `Any` as type? Can any of them have another type than `str` or `None`? If not, then I suggest `Optional[str]` instead of `Any` (or simply `str` for those who cannot be `None`).